2024-07-05 18:50:23 UTC
macOS-10.16-x86_64-i386-64bit
Ren'Py 8.2.2.24061201

Early init took 0.10s
Loading error handling took 0.25s
Loading script took 4.37s
Loading save slot metadata took 0.11s
Loading persistent took 0.00s
Running init code took 0.17s
Loading analysis data took 0.09s
Analyze and compile ATL took 0.01s
Reloading save slot metadata took 0.07s
Index archives took 0.00s
Backing up script files to '/Users/tes/Library/RenPy/backups/KTPVisualNovel':
Dump and make backups took 0.12s
Cleaning cache took 0.00s
Making clean stores took 0.00s
Initial gc took 0.13s
DPI scale factor: 1.000000
Creating interface object took 0.00s
Cleaning stores took 0.00s
Init translation took 0.19s
Build styles took 0.01s
Load screen analysis took 0.00s
Analyze screens took 0.11s
Save screen analysis took 0.06s
Prepare screens took 0.26s
Save pyanalysis. took 0.05s
Save bytecode. took 0.12s
Running _start took 0.01s
Interface start took 0.49s

Initializing gl2 renderer:
primary display bounds: (0, 0, 1440, 900)
swap interval: 1 frames
Windowed mode.
Vendor: "b'Intel Inc.'"
Renderer: b'Intel(R) UHD Graphics 617'
Version: b'2.1 INTEL-22.5.11'
Display Info: None
Screen sizes: virtual=(1920, 1080) physical=(1440, 771) drawable=(2880, 1542)
Maximum texture size: 4096x4096
Screen sizes: virtual=(1920, 1080) physical=(1440, 771) drawable=(2880, 1542)
Maximum texture size: 4096x4096
Screen sizes: virtual=(1920, 1080) physical=(1440, 771) drawable=(2880, 1800)
Maximum texture size: 4096x4096
Screen sizes: virtual=(1920, 1080) physical=(1440, 900) drawable=(2880, 1800)
Maximum texture size: 4096x4096

Full traceback:
  File "renpy/common/_layout/screen_main_menu.rpym", line 28, in script
    python hide:
  File "/Users/tes/Documents/Fanbase/renpy/renpy/ast.py", line 823, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/Users/tes/Documents/Fanbase/renpy/renpy/python.py", line 1178, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "renpy/common/_layout/screen_main_menu.rpym", line 28, in <module>
    python hide:
  File "renpy/common/_layout/screen_main_menu.rpym", line 35, in _execute_python_hide
    ui.interact()
  File "/Users/tes/Documents/Fanbase/renpy/renpy/ui.py", line 301, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "/Users/tes/Documents/Fanbase/renpy/renpy/display/core.py", line 2166, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "/Users/tes/Documents/Fanbase/renpy/renpy/display/core.py", line 2738, in interact_core
    renpy.audio.audio.interact()
  File "/Users/tes/Documents/Fanbase/renpy/renpy/audio/audio.py", line 1245, in interact
    periodic()
  File "/Users/tes/Documents/Fanbase/renpy/renpy/audio/audio.py", line 1199, in periodic
    raise_(exc[0], exc[1], exc[2])
  File "lib/python3.9/future/utils/__init__.py", line 444, in raise_
  File "/Users/tes/Documents/Fanbase/renpy/renpy/audio/audio.py", line 1177, in periodic_thread_main
    periodic_pass()
  File "/Users/tes/Documents/Fanbase/renpy/renpy/audio/audio.py", line 1119, in periodic_pass
    c.periodic()
  File "/Users/tes/Documents/Fanbase/renpy/renpy/audio/audio.py", line 528, in periodic
    topf = load(filename)
  File "/Users/tes/Documents/Fanbase/renpy/renpy/audio/audio.py", line 72, in load
    rv = renpy.loader.load(fn, directory="audio")
  File "/Users/tes/Documents/Fanbase/renpy/renpy/loader.py", line 672, in load
    raise IOError("Couldn't find file '%s'." % name)
OSError: Couldn't find file 'audio/Dreamcatcher_inst_v2.mp3'.

While running game code:
OSError: Couldn't find file 'audio/Dreamcatcher_inst_v2.mp3'.
Resetting cache.
Early init took 177.63s
Loading error handling took 0.09s
Loading script took 0.66s
Loading save slot metadata took 0.09s
Loading persistent took 0.00s
Running init code took 0.09s
Loading analysis data took 0.05s
Analyze and compile ATL took 0.01s
Reloading save slot metadata took 0.07s
Index archives took 0.00s
Dump and make backups took 0.07s
Cleaning cache took 0.00s
Making clean stores took 0.00s
Initial gc took 0.17s
DPI scale factor: 1.000000
Creating interface object took 0.00s
Cleaning stores took 0.00s
Init translation took 0.17s
Build styles took 0.01s
Load screen analysis took 0.08s
Analyze screens took 0.00s
Save screen analysis took 0.00s
Prepare screens took 0.16s
Save pyanalysis. took 0.00s
Save bytecode. took 0.00s
Running _start took 0.00s
Interface start took 0.13s

Initializing gl2 renderer:
primary display bounds: (0, 0, 1440, 900)
swap interval: 1 frames
Windowed mode.
Vendor: "b'Intel Inc.'"
Renderer: b'Intel(R) UHD Graphics 617'
Version: b'2.1 INTEL-22.5.11'
Display Info: None
Screen sizes: virtual=(1920, 1080) physical=(1440, 900) drawable=(2880, 1800)
Maximum texture size: 4096x4096
Screen sizes: virtual=(1920, 1080) physical=(1440, 900) drawable=(2880, 1800)
Maximum texture size: 4096x4096

Full traceback:
  File "renpy/common/00start.rpy", line 216, in script call
    call _load_reload_game from _call__load_reload_game_1
  File "renpy/common/00keymap.rpy", line 531, in script
    python hide:
  File "/Users/tes/Documents/Fanbase/renpy/renpy/ast.py", line 823, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/Users/tes/Documents/Fanbase/renpy/renpy/python.py", line 1178, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "renpy/common/00keymap.rpy", line 531, in <module>
    python hide:
  File "renpy/common/00keymap.rpy", line 542, in _execute_python_hide
    renpy.load(renpy.session["_reload_slot"])
  File "/Users/tes/Documents/Fanbase/renpy/renpy/loadsave.py", line 829, in load
    log.unfreeze(roots, label="_after_load")
  File "/Users/tes/Documents/Fanbase/renpy/renpy/rollback.py", line 1130, in unfreeze
    self.rollback(0, force=True, label=label, greedy=greedy, on_load=True)
  File "/Users/tes/Documents/Fanbase/renpy/renpy/rollback.py", line 925, in rollback
    self.load_failed()
  File "/Users/tes/Documents/Fanbase/renpy/renpy/rollback.py", line 843, in load_failed
    raise Exception("Couldn't find a place to stop rolling back. Perhaps the script changed in an incompatible way?")
Exception: Couldn't find a place to stop rolling back. Perhaps the script changed in an incompatible way?

While running game code:
  File "renpy/common/00start.rpy", line 216, in script call
    call _load_reload_game from _call__load_reload_game_1
  File "renpy/common/00keymap.rpy", line 531, in script
    python hide:
  File "renpy/common/00keymap.rpy", line 531, in <module>
    python hide:
  File "renpy/common/00keymap.rpy", line 542, in _execute_python_hide
    renpy.load(renpy.session["_reload_slot"])
Exception: Couldn't find a place to stop rolling back. Perhaps the script changed in an incompatible way?
Resetting cache.
Early init took 190.62s
Loading error handling took 0.08s
Loading script took 0.33s
Loading save slot metadata took 0.07s
Loading persistent took 0.00s
Running init code took 0.05s
Loading analysis data took 0.04s
Analyze and compile ATL took 0.01s
Reloading save slot metadata took 0.04s
Index archives took 0.00s
Dump and make backups took 0.07s
Cleaning cache took 0.00s
Making clean stores took 0.00s
Initial gc took 0.13s
DPI scale factor: 1.000000
Creating interface object took 0.00s
Cleaning stores took 0.00s
Init translation took 0.10s
Build styles took 0.01s
Load screen analysis took 0.09s
Analyze screens took 0.00s
Save screen analysis took 0.00s
Prepare screens took 0.19s
Save pyanalysis. took 0.00s
Save bytecode. took 0.00s
Running _start took 0.00s
Interface start took 0.13s

Initializing gl2 renderer:
primary display bounds: (0, 0, 1440, 900)
swap interval: 1 frames
Windowed mode.
Vendor: "b'Intel Inc.'"
Renderer: b'Intel(R) UHD Graphics 617'
Version: b'2.1 INTEL-22.5.11'
Display Info: None
Screen sizes: virtual=(1920, 1080) physical=(1440, 900) drawable=(2880, 1800)
Maximum texture size: 4096x4096
Screen sizes: virtual=(1920, 1080) physical=(1440, 900) drawable=(2880, 1800)
Maximum texture size: 4096x4096

Full traceback:
  File "renpy/common/00start.rpy", line 216, in script call
    call _load_reload_game from _call__load_reload_game_1
  File "renpy/common/00keymap.rpy", line 531, in script
    python hide:
  File "/Users/tes/Documents/Fanbase/renpy/renpy/ast.py", line 823, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/Users/tes/Documents/Fanbase/renpy/renpy/python.py", line 1178, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "renpy/common/00keymap.rpy", line 531, in <module>
    python hide:
  File "renpy/common/00keymap.rpy", line 542, in _execute_python_hide
    renpy.load(renpy.session["_reload_slot"])
  File "/Users/tes/Documents/Fanbase/renpy/renpy/loadsave.py", line 829, in load
    log.unfreeze(roots, label="_after_load")
  File "/Users/tes/Documents/Fanbase/renpy/renpy/rollback.py", line 1130, in unfreeze
    self.rollback(0, force=True, label=label, greedy=greedy, on_load=True)
  File "/Users/tes/Documents/Fanbase/renpy/renpy/rollback.py", line 925, in rollback
    self.load_failed()
  File "/Users/tes/Documents/Fanbase/renpy/renpy/rollback.py", line 843, in load_failed
    raise Exception("Couldn't find a place to stop rolling back. Perhaps the script changed in an incompatible way?")
Exception: Couldn't find a place to stop rolling back. Perhaps the script changed in an incompatible way?

While running game code:
  File "renpy/common/00start.rpy", line 216, in script call
    call _load_reload_game from _call__load_reload_game_1
  File "renpy/common/00keymap.rpy", line 531, in script
    python hide:
  File "renpy/common/00keymap.rpy", line 531, in <module>
    python hide:
  File "renpy/common/00keymap.rpy", line 542, in _execute_python_hide
    renpy.load(renpy.session["_reload_slot"])
Exception: Couldn't find a place to stop rolling back. Perhaps the script changed in an incompatible way?
